src/views/AlertTable/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { useMemo } from 'react';
+import {
+    gql,
+    useQuery,
+} from '@apollo/client';
+import {
+    Container,
+    Pager,
+    Table,
+} from '@ifrc-go/ui';
+import { SortContext } from '@ifrc-go/ui/contexts';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import {
+    createBooleanColumn,
+    createStringColumn,
+} from '@ifrc-go/ui/utils';
+
+import {
+    PublicAlertTypeQuery,
+    PublicAlertTypeQueryVariables,
+} from '#generated/types';
+import useFilterState from '#hooks/useFilterState';
+import { createLinkColumn } from '#utils/domain/tableHelpers';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type AlertListItem = {
+    sent: boolean;
+    id: number;
+    event: string | null | undefined,
+    eventCategory: string;
+    region: string;
+    countries_details: string[];
+    admin: string;
+};
+const alertKeySelector = (item: AlertListItem) => item.id;
+
+const ALERT_TYPE = gql`
+query  AlertType {
+    public {
+      alertInfos {
+        items {
+          event
+          category
+        }
+      }
+      region(pk: "1") {
+        id
+        name
+      }
+      country(pk: "1") {
+        id
+        name
+      }
+      admin1s {
+        items {
+          id
+          name
+        }
+      }
+      alert(pk: "2") {
+        sent
+        url
+      }
+    }
+  }
+`;
```

Comment:
```suggestion
query  AlertType (
   $regionId: ID!,
   $countryId: ID!,
   $alertId: ID!,
    ) {
    public {
      alertInfos {
        items {
          event
          category
        }
      }
      region(pk: $regionId) {
        id
        name
      }
      country(pk: $countryId) {
        id
        name
      }
      admin1s {
        items {
          id
          name
        }
      }
      alert(pk: $alertId) {
        sent
        url
      }
    }
  }
`;
```
Pass the Id in the query.